### DISCUSSION TITLE
总是爆内存 听不了几首歌 是不是只有我一个

### DISCUSSION THREAD

**Discussion by xiaopebaka** [1715173329/UnblockNeteaseMusic#235] [General]
用的树莓派 我还给了2个G的虚拟内存 都不行
<img width="874" alt="image" src="https://user-images.githubusercontent.com/57299718/135570681-d9182aa9-d5f5-4235-8573-815310c686dc.png">

**Comment by 1715173329** [comment]
NO_CACHE=true 关掉缓存试试

**Comment by pan93412** [comment]
> 用的树莓派 我还给了2个G的虚拟内存 都不行
> <img width="874" alt="image" src="https://user-images.githubusercontent.com/57299718/135570681-d9182aa9-d5f5-4235-8573-815310c686dc.png">
> 

如果要開 Cache，你可能還得調 Node.js 的 Heap Size。不太清楚發生的實際原因。

**Comment by gavin8282** [comment]
> 用的树莓派 我还给了2个G的虚拟内存 都不行
> <img alt="image" width="874" src="https://user-images.githubusercontent.com/57299718/135570681-d9182aa9-d5f5-4235-8573-815310c686dc.png">

我也在树莓派上跑着一个，还是起的lxc，一直很稳定。建议使用64位的lite版本[http://mirrors.ustc.edu.cn/raspberry-pi-os-images/raspios_lite_arm64/images/raspios_lite_arm64-2021-05-28/2021-05-07-raspios-buster-arm64-lite.zip](url)

**Comment by xiaopebaka** [comment]
> NO_CACHE=true 关掉缓存试试
![5$TS{SRDN_(PB{7%W~2O HY](https://user-images.githubusercontent.com/57299718/136212935-54fcbb7a-a9a9-4922-b471-71ebae206933.png)
好像也不行

**Comment by rampageX** [comment]
N1 跑的 armbian aarch64 没啥问题：

`2129   49.22 MB /usr/local/bin/node /root/src/UnblockNeteaseMusic/app.js -p 8880:8881`
